src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts

```diff
@@ -151,7 +151,10 @@ const $proxyWalletForProxied = combine(
   { skipVoid: false },
 );
 
-const $hasProxies = combine($proxyAccounts, (accounts) => accounts.length > 0);
+const $hasProxies = combine(
+  { accounts: $proxyAccounts, groups: proxyModel.$walletsProxyGroups, wallet: walletSelectModel.$walletForDetails },
```

Comment:
Please make each statement on it's own line for readability
and make oneliner arrow function with curly brackets & `return`